Angelus/Bliss/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/Angelus.06_ROS.Bliss.v2_16-9.jpg DESTINATION reactos/Web/Wallpaper FOR all)
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/Angelus.06_ROS.Bliss.v2_4-3.jpg DESTINATION reactos/Web/Wallpaper FOR all)
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/Angelus.06_ROS.Bliss_16-9_1365x768.jpg DESTINATION reactos/Web/Wallpaper FOR all)
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/Angelus.06_ROS.Bliss_4-3_1024x768.jpg DESTINATION reactos/Web/Wallpaper FOR all)
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/CristalROS.Bliss.16x10.jpg DESTINATION reactos/Web/Wallpaper FOR all)
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/CristalROS.Bliss.16x9.jpg DESTINATION reactos/Web/Wallpaper FOR all)
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/CristalROS.Bliss.4x3.jpg DESTINATION reactos/Web/Wallpaper FOR all)
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/Dream.Bliss.16x10.jpg DESTINATION reactos/Web/Wallpaper FOR all)
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/Dream.Bliss.16x9.jpg DESTINATION reactos/Web/Wallpaper FOR all)
+add_cd_file(FILE ${CMAKE_CURRENT_SOURCE_DIR}/Dream.Bliss.4x3.jpg DESTINATION reactos/Web/Wallpaper FOR all)
```

Comment:
Do we still need any of those CMakeLists.txt files?

Reply:
We will be able to restore the previous directory tree by copying to r`eactos/modules/wallpapers` anytime.

Reply:
ok